src/Headroom.js

```diff
@@ -160,11 +160,17 @@ Headroom.prototype = {
   },
 
   addClass: function(className) {
-    this.elem.classList.add(this.classes[className]);
+    this.elem.classList.add.apply(
+      this.elem.classList,
+      this.classes[className].split(" ")
+    );
   },
 
   removeClass: function(className) {
-    this.elem.classList.remove(this.classes[className]);
+    this.elem.classList.remove.apply(
+      this.elem.classList,
+      this.classes[className].split(" ")
+    );
   },
 
   hasClass: function(className) {
```

Comment:
We also need changes to `hasClass`, so it checks that each of the (multiple) classes is present. If you add a test like this you will see the current implementation fails:

```js
  it("handles multiple classes", () => {
    initialiseHeadroom({
      classes: {
        pinned: "headroom--pinned foo",
        unpinned: "headroom--unpinned bar"
      }
    });

    cy.scrollTo(0, 50);
    cy.get("header")
      .should("not.be.pinned")
      .should("have.class", "bar");

    cy.scrollTo(0, 25);
    cy.get("header")
      .should("be.pinned")
      .should("have.class", "foo");
  });
```

But if you change to this, you should see the tests now pass:

```js
  hasClass: function(className) {
    return this.classes[className].split(" ").every(function(cls) {
      return this.classList.contains(cls);
    }, this.elem);
  },
```

Reply:
`hasClass` is only used by the library itself and always with a single class. Do we really need to touch it?

Reply:
Yes, it's a required change. Can you add the test above to this PR? You will see the failure then, and we can catch any future regressions.

The reason being that (unfortunately) some state is stored in the DOM, and we use `hasClass` to check whether a state transition has occurred (e.g. pinned -> unpinned). Without modifying `hasClass`, if you supply multiple classes then a call like `this.elem.classList.contains("unpinned foo bar")` would be made, which would return false. Thus we need to check for the existence of every class for each state.

Reply:
Just pushed a couple commits with the changes